notificationservice/src/main/java/com/example/notificationservice/models/requestbody/BlacklistRequestBody.java

```diff
@@ -0,0 +1,15 @@
+package com.example.notificationservice.models.requestbody;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class BlacklistRequestBody {
```

Comment:
You can add this annotation here if the payload is coming in snake case
@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)

Reply:
You wouldn't need @JsonProperty for each field then